ui/ember-cli-build.js

```diff
@@ -89,10 +90,32 @@ module.exports = function (defaults) {
   app.import('node_modules/jsondiffpatch/dist/jsondiffpatch.umd.js');
   app.import('node_modules/jsondiffpatch/dist/formatters-styles/html.css');
 
-  app.import('app/styles/bulma/bulma-radio-checkbox.css');
-
   app.import('node_modules/@hashicorp/structure-icons/dist/loading.css');
   app.import('node_modules/@hashicorp/structure-icons/dist/run.css');
 
-  return app.toTree();
+  //return app.toTree();
+  const { Webpack } = require('@embroider/webpack');
+  return require('@embroider/compat').compatBuild(app, Webpack, {
+    staticAddonTestSupportTrees: true,
+    staticAddonTrees: true,
+    staticModifiers: true,
+    //staticHelpers: true,
+    // staticComponents: true,
+    // staticEmberSource: true,
+    // splitAtRoutes: ['route.name'], // can also be a RegExp
```

Comment:
I didn't try turning on `staticHelpers` and `staticComponents`, but they will require the most work to enable (and they both need to be enabled at the same time). `staticEmberSource` likely needs Ember 5.x to try out (that's what we needed to turn the flag on in HCP).